pkg/cmd/util/clientcmd/factory.go

```diff
@@ -207,23 +211,14 @@ func (f *Factory) ApproximatePodTemplateForObject(object runtime.Object) (*api.P
 		return fallback, nil
 
 	default:
-		pod, err := f.AttachablePodForObject(object, 1*time.Minute)
```

Comment:
Why this minute timeout doesn't have any effect, in the original issue Clayton pointed out?